src/mbgl/text/placement.cpp

```diff
@@ -165,10 +165,12 @@ void Placement::placeLayerBucket(
                 placements.erase(symbolInstance.crossTileID);
             }
             
-            placements.emplace(symbolInstance.crossTileID, JointPlacement(placeText, placeIcon, offscreen));
+            placements.emplace(symbolInstance.crossTileID, JointPlacement(placeText, placeIcon, offscreen || bucket.justReloaded));
```

Comment:
Same as in JS: I think we should rename "offscreen" from this point forward since it combines two concepts now.

Reply:
👍

Reply:
👍